Hmm, `#if __has_attribute(swiftasynccall) && (__has_feature(swiftasynccc) || __has_extension(swiftasynccc))` is not acceptable with MSVC cl.exe?

https://ci-external.swift.org/job/swift-PR-windows/23188/console
```
C:\Users\swift-ci\jenkins\workspace\swift-PR-windows\swift\include\swift/Runtime/Config.h(231): fatal error C1012: unmatched parenthesis: missing ')'
```

```c
  // SWIFT_CC(swiftasync) is the Swift async calling convention.
  // We assume that it supports mandatory tail call elimination.
  #if __has_attribute(swiftasynccall) && (__has_feature(swiftasynccc) || __has_extension(swiftasynccc))
  #define SWIFT_CC_swiftasync __attribute__((swiftasynccall))
  #else
  #define SWIFT_CC_swiftasync SWIFT_CC_swift
  #endif
```

--- a/include/swift/Runtime/Config.h
+++ b/include/swift/Runtime/Config.h
@@ -228,8 +228,12 @@ extern uintptr_t __COMPATIBILITY_LIBRARIES_CANNOT_CHECK_THE_IS_SWIFT_BIT_DIRECTL
 
 // SWIFT_CC(swiftasync) is the Swift async calling convention.
 // We assume that it supports mandatory tail call elimination.
-#if __has_feature(swiftasynccc) && __has_attribute(swiftasynccall)
-#define SWIFT_CC_swiftasync __attribute__((swiftasynccall))
+#if __has_attribute(swiftasynccall)
+# if __has_feature(swiftasynccc) || __has_extension(swiftasynccc)
+#  define SWIFT_CC_swiftasync __attribute__((swiftasynccall))
+# else
+#  define SWIFT_CC_swiftasync SWIFT_CC_swift
+# endif
 #else
 #define SWIFT_CC_swiftasync SWIFT_CC_swift
 #endif

